demo/modal.py

```diff
@@ -0,0 +1,113 @@
+"""Simple modal that looks similar to Angular Component Dialog."""
+
+import mesop as me
+
+
+@me.stateclass
+class State:
+  is_open: bool = False
+
+
+@me.page(
+  security_policy=me.SecurityPolicy(
+    allowed_iframe_parents=["https://google.github.io"]
+  ),
+  path="/modal",
+)
+def app():
+  state = me.state(State)
+
+  with modal(state.is_open):
+    with modal_container():
```

Comment:
That's fair. Since this is a copyable example I think your suggestion makes sense. I had wanted to keep it separate in case someone wanted a different design for the modal. So if they didn't want the rounded corners. But they can just edit the code in this case.